static/partials/page.html

```diff
@@ -1,6 +1,8 @@
 <div id="page" class="ng-cloak">
 	<div ng-if="page" ng-include="page"></div>
 	<div ng-hide="page" id="help">
-		<usability/>
+		<div class="help">
+			<usability/>
+		</div>
 	</div>
```

Comment:
The directive is set to replace but I could add it to the [first div](https://github.com/strukturag/spreed-webrtc/blob/master/static/partials/usability.html#L1) in `usability.html`.